src/main/java/uk/co/evoco/webdriver/utils/SelectBoxUtils.java

```diff
@@ -45,42 +49,46 @@ public static void itemByVisibleText(WebElement selectBox, String visibleText) {
     }
 
     /**
-     *
      * @param webElement active WebElement, already located
-     * @param htmlValue HTML value attribute
-     * @param timeout time in seconds to keep trying
+     * @param htmlValue  HTML value attribute
+     * @param timeout    time in seconds to keep trying
      * @throws Throwable any unhandled or un-tolerated exception
      */
-    public static void tolerantItemByHtmlValueAttribute(WebElement webElement,  String htmlValue, int timeout)
+    public static void tolerantItemByHtmlValueAttribute(WebElement webElement, String htmlValue, int timeout)
             throws Throwable {
         new SendKeysUtils().tolerantInteraction(
                 webElement, SelectBoxInteractionType.BY_VALUE, Optional.of(htmlValue), Optional.empty(), timeout);
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes